newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -225,6 +279,70 @@ export const GameDetailsDialog = ({
             </Line>
           </ColumnStackLayout>
         ) : null}
+        {currentTab === 'builds' ? (
```

Comment:
I could start extracting this to its own component, but it's quite simple for the moment (may allow build deletion in the future...)

Reply:
I think that's fine for now, we can extract when needed, or now if you feel like it.